et_ee.lang

```diff
@@ -0,0 +1,409 @@
+gui.xaero_entity_info_potion_effects=Võlujoogimõjud (ainult üksikmäng)
+gui.xaero_entity_info_potion_effects_scale=Võlujoogimõju suurus
+gui.xaero_terrain_depth=Maastiku sügavus
+gui.xaero_terrain_slopes=Maastiku kalded
+gui.xaero_block_transparency=Ploki läbipaistvus
+gui.xaero_team=Meeskond
+gui.xaero_world_server=Maailm/server
+gui.xaero_subworld_dimension=Alam-maailm/dimensioon
+gui.xaero_switch_waypoint_set=Lülita teepunkti kogumite vahel
+gui.xaero_waypoint_opacity_ingame=Teepunkti mängusisene läbipaistmatus
+gui.xaero_waypoint_opacity_map=Teepunkti kaardipealne läbipaistmatus
+gui.xaero_hide_world_names=Peida IPd/maailma nimed
+gui.xaero_open_slime=Ava klahvivajutusel
+gui.xaero_options=Valikud
+gui.xaero_make_automatic=Tee maailm/server autom.
+gui.xaero_make_automatic_msg1=Kas oled kindel?
+gui.xaero_make_automatic_msg2=See vahetab kõigi valitud maailma/serveri teepunktide andmed automaatse vastu, simuleerides valitud maailma/serveri automaatseks tegemist. Veendu, et tead, mida teed.
+gui.xaero_make_multi_automatic=Tee alam-maailm autom.
+gui.xaero_make_multi_automatic_msg1=Kas oled kindel?
+gui.xaero_make_multi_automatic_msg2=See vahetab kõigi valitud alam-maailma teepunktide andmed automaatse vastu, simuleerides valitud alam-maailma automaatseks tegemist. Veendu, et tead, mida teed.
+gui.xaero_delete_world=Kustuta maailm/server
+gui.xaero_delete_world_msg1=Kas soovid kustutada kõik teepunktide andmed valitud maailma/serveri jaoks?
+gui.xaero_delete_world_msg2=Seda tegevust ei saa tagasi võtta! Veendu, et tead, mida teed.
+gui.xaero_delete_multi_world=Kustuta alam-maailm
+gui.xaero_delete_multi_world_msg1=Kas soovid kustutada kõik teepunktide andmed valitud alam-maailma jaoks?
+gui.xaero_delete_multi_world_msg2=Seda tegevust ei saa tagasi võtta! Veendu, et tead, mida teed.
+gui.xaero_always_show_distance=Alati kauguse teepunktini kuvamine
+gui.xaero_item_tooltip=Eseme hüpiktekst
+gui.xaero_entity_tooltip_settings=Eseme hüpikteksti seaded
+gui.xaero_item_tooltip_min_lines=Minimaalselt ridu
+gui.xaero_item_tooltip_time=Kuvamisaeg
+gui.xaero_multiply_all_by_8=Alam-maailma koordinaadid * 8
+gui.xaero_divide_all_by_8=Alam-maailma koordinaadid / 8
+gui.xaero_multiply_msg1=Seda valikut peaks kasutama vaid ebakorrektsete teepunkti koordinaatide parandamiseks.
+gui.xaero_multiply_msg2=Kas oled kindel, et soovid kõik alam-maailma koordinaadid KORRUTADA 8-ga?
+gui.xaero_divide_msg2=Kas oled kindel, et soovid kõik alam-maailma koordinaadid JAGADA 8-ga?
+gui.xaero_share=Jaga
+gui.xaero_share_msg1=Kas soovid kindlasti jagada seda teepunkti vestluses §cKÕIGIGA§f?
+gui.xaero_share_msg2=Veendu, et sa ei paljasta salajast asukohta!
+gui.xaero_roof=lagi
+gui.xaero_partial=osaliselt
+gui.xaero_full=täielikult
+gui.xaero_show_light_level=Valgustaseme kuvamine
+gui.xaero_up=[Mine üles]
+gui.xaero_down=[Mine alla]
+gui.xaero_display_all_sets=Lülita kõigi teepunktide kogumite nähtavus
+gui.xaero_waypoint_name=teepunkti nimi
+gui.xaero_set_name=kogumi nimi
```

Comment:
Should probably be capitalized

Reply:
I was wondering, where is it displayed? If it's in the textbox, it might be better not capitalized, like _yaw_ and _initials_ are.

Reply:
Oh. I think it's a line that isn't displayed anymore. Used to be displayed above the text field when creating a new set. I might bring it back when/if the set creation screen gets more options.